cmd/ethrex/cli.rs

```diff
@@ -273,8 +273,8 @@ impl Subcommand {
     }
 }
 
-pub fn remove_db(datadir: &str, force: bool) {
-    let data_dir = set_datadir(datadir);
+pub fn remove_db(datadir: &str, force: bool, network: &Option<String>) {
```

Comment:
for simplicity, maybe remove_db should just remove everything, not just the db of the current network.